content/content/oop_macro.md

```diff
@@ -43,7 +43,7 @@ To get that nice notation, we can use a macro:
 ```nimrod
 import macros
 
-macro class*(head: expr, body: stmt): stmt {.immediate.} =
+macro class*(head, body): untyped =
   # The macro is immediate so that it doesn't
```

Comment:
This comment now requires further explaination since `{.immediate.}` is gone. See [Typed vs untyped parameters](http://nim-lang.org/docs/manual.html#templates-typed-vs-untyped-parameters) in the manual:

> A template where every parameter is untyped is called an immediate template. For historical reasons templates can be explicitly annotated with an immediate pragma and then these templates do not take part in overloading resolution and the parameters' types are ignored by the compiler. Explicit immediate templates are now deprecated.